server/fleet/global_policies.go

```diff
@@ -11,6 +11,9 @@ type Policy struct {
 	UpdateCreateTimestamps
 }
 
-func (Policy) AuthzType() string {
+func (p Policy) AuthzType() string {
+	if p.TeamID != nil {
+		return "team_policy"
+	}
```

Comment:
I wonder if it's better to do this nil check in the `policy.rego` and have a single `policy` `AuthzType`?

Reply:
Sounds reasonable, I'll give it a try.